source/our-stacks/android/index.md

```diff
@@ -1,6 +1,63 @@
 ---
 title: Android
-date: 2016-09-29 15:27:11
+date: 2016-10-06 17:37:40
 layout: stack
 ---
-Total Android
\ No newline at end of file
+
+Android developers at Nodes strive to build awesome apps using the Material Design Guidelines and Google provided design tools. 
+
+Custom designs are important to Nodes and helping the client get the best possible Android experience is a high priority for the Android devs.
+
+### Tasks
+- Maintain internal frameworks  
+- Take over code bases and adapt/update to newest patterns
+- Implement and tweak iOS designs to the _true Android way_
+- Help the UX and UI team with the ever-changing design language on Android
+- Estimate changes and feature requests on ongoing projects
+- Refactor common used code into libraries/utilities
+- Work closely together with the QA team
+
+## Stack
+
+### Language
+
+##### Java
+Java 1.0 was released in 1995 and is still being used widely today - in part because of Android. Being an old language the toolchain is mature, and features like annotation processing, instant-run and an excellent Java SDK provides a nice toolset for development.
+
+### IDE
+Android Studio is the natural choice on the platform with all the bells and whistles it brings. 
```

Comment:
fixed in https://github.com/nodesagency/blog/pull/18/commits/35eeec79790f927f60d27f1c1f234742a106753f